vispy/gloo/glir.py

```diff
@@ -259,19 +260,25 @@ def as_es2_command(command):
 class BaseGlirParser(object):
     """ Base clas for GLIR parsers that can be attached to a GLIR queue.
     """
-    
+
+    def __init__(self):
+        self.capabilities = dict(
```

Comment:
Maybe -- I think this is what @almarklein had in mind (see #669) and this seemed like the simplest place to put it, but I could be wrong.